app/src/main/java/org/mifos/selfserviceapp/api/ApiEndPoints.java

```diff
@@ -12,4 +12,5 @@
     public static final String AUTHENTICATION = "authentication";
     public static final String CLIENTS = "clients";
     public static final String SAVINGSACOUNTS = "savingsaccounts";
+    public static final String ACCOUNTTRANSFERS = "accounttransfers";
```

Comment:
Completely agree. Mistake at my part. Will change it.

On Aug 24, 2016 9:11 PM, "Satya Narayan" <notifications@github.com> wrote:

> In app/src/main/java/org/mifos/selfserviceapp/api/ApiEndPoints.java
> <https://github.com/openMF/self-service-app/pull/17#discussion_r76082104>:
>
> > @@ -12,4 +12,5 @@
> >      public static final String AUTHENTICATION = "authentication";
> >      public static final String CLIENTS = "clients";
> >      public static final String SAVINGSACOUNTS = "savingsaccounts";
> > +    public static final String ACCOUNTTRANSFERS = "accounttransfers";
>
> usually the style here is ACCOUNT_TRANSFERS which makes the name easier to
> read. without the _ it's a little difficult to understand.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openMF/self-service-app/pull/17/files/d657bda5ccb13f14d37e46724ec5d2a6b347c669#r76082104>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AK4JNnj5m-Z-oIHJY-ANaVunTsTPuBkkks5qjGYggaJpZM4JqhyH>
> .
>